numpy/core/src/umath/ufunc_object.c

```diff
@@ -479,6 +479,26 @@ _is_alnum_underscore(char ch)
     return _is_alpha_underscore(ch) || (ch >= '0' && ch <= '9');
 }
 
+/*
+ * Convert a string into a number
+ */
+ static npy_intp
+ _get_size(const char* str)
+ {
```

Comment:
Remote the leaing space on these three lines (and the last one)

Reply:
fixed